app/views/candidates/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1 style="text-align:center">Candidates</h1>
+<div class="row">
+  <div class="col-sm-3">
+    <div style="text-align:center">
+      <a href="/candidates/new" >
+        <%= image_tag "plus_sign.png"  %>
+      </a>
+    </div>
+  </div>
+  <% @candidates.each do |candidate|%>
+    <div class="col-sm-3">
+      <div class="card m-2" style="width: 15rem;">
+        <img src="https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-chat/ava1-bg.webp" alt="Avatar" class="img-fluid my-5" style="width: 80px;" />
+        <div class="card-body">
+          <h5 class="card-title"><%= candidate.name %></h5>
+          <%= link_to 'View Candidate', candidate_path(candidate.id), class: "nav-link active" %>
```

Comment:
suggestion: do we need to write `candidate_path(candidate.id)`, it should work just with passing candidate object like this `<%= link_to 'View Candidate', candidate_path(candidate, class: "nav-link active" %>`